packages/catalog-process/open-api/catalog-service-spec.yml

```diff
@@ -630,6 +630,53 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
+  /eservices/{eServiceId}/descriptors/{descriptorId}/update:
+    parameters:
+      - $ref: "#/components/parameters/CorrelationIdHeader"
+    post:
+      security:
+        - bearerAuth: []
+      tags:
+        - process
+      summary: Update the selected descriptor.
+      operationId: updateDescriptor
+      parameters:
+        - name: eServiceId
+          in: path
+          description: the eservice id
+          required: true
+          schema:
+            type: string
+            format: uuid
+        - name: descriptorId
+          in: path
+          description: the descriptor Id
+          required: true
+          schema:
+            type: string
+            format: uuid
+      requestBody:
+        description: Payload
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/UpdateEServiceDescriptorQuotasSeed"
+      responses:
+        "200":
```

Comment:
Updated.
- We need to return 200 because on Scala we are returning 200. If we change the status code here, we would break QA tests
- The object returned in the Scala implementation is the entire eservice updated. We are not following this pattern here in the monorepo, because we never return the object based on the event data. So here we could just return an id. (This doesn't break QA tests)